Security_Engine/Compute/CameraFieldOfView.cs

```diff
@@ -129,7 +129,10 @@ public static PolyCurve CameraFieldOfView(this CameraDevice cameraDevice, List<P
             //create points chain
             List<Point> pointsChain = linesDict.PointsChain(cameraLocation, radius, distanceTolerance);
 
-            //create cone
+            //reverse points chain to have correct start and end arc angles
+            pointsChain.Reverse();
```

Comment:
Shouldn't that be a part of `Query.PointsChain` method?

Reply:
Yes, I will move that 👍